build.gradle

```diff
@@ -51,24 +52,52 @@ allprojects {
         mavenCentral()
     }
 
+    configurations {
+        ajc
+        aspects
+        compile {
+            extendsFrom aspects
+        }
+    }
+
     dependencies {
         // MongoDB
+        ajc "org.aspectj:aspectjtools:$aspectjVersion"
+        implementation "org.aspectj:aspectjrt:$aspectjVersion"
         implementation group: 'org.mongodb', name: 'mongodb-driver-sync', version: mongodbDriverVersion
         implementation group: 'org.mongodb', name: 'mongodb-driver-core', version: mongodbDriverVersion
         implementation group: 'org.mongodb', name: 'mongodb-driver', version: mongodbDriverVersion
         implementation group: 'com.google.guava', name: 'guava', version: guavaVersion
         implementation group: 'org.apache.commons', name: 'commons-lang3', version: lang3Version
+        ajc "org.aspectj:aspectjtools:$aspectjVersion"
         testCompile group: 'junit', name: 'junit', version: junitVersion
 
-
-
         // Test
         testImplementation group: 'org.junit.jupiter', name: 'junit-jupiter-api', version: junitJupiterVersion
         testRuntimeOnly  group: 'org.junit.jupiter', name: 'junit-jupiter-engine', version: junitJupiterVersion
         testCompile  group: 'org.mockito', name: 'mockito-core', version: mockitoVersion
         testCompile  group: 'org.mockito', name: 'mockito-junit-jupiter', version: mockitoVersion
     }
 
+    ant.lifecycleLogLevel = AntBuilder.AntMessagePriority.VERBOSE
```

Comment:
When I run a build of this PR I'm getting a TON of messages that seem to be related to ant, I suspect it's related to this line being VERBOSE?  If so can we change this, the log lines I'm seeing look like the following:
```
[ant:iajc] Join point 'method-execution(java.lang.Object com.mongodb.jdbc.MongoConnection.unwrap(java.lang.Class))' in Type 'com.mongodb.jdbc.MongoConnection' (MongoConnection.java:517) advised by afterThrowing advice from 'com.mongodb.jdbc.logging.LoggingAspect' (LoggingAspect.aj:72) [with runtime test]
```

Reply:
That works, thanks!